hydromt/models/model.py

```diff
@@ -54,22 +47,11 @@ class Model(object, metaclass=ABCMeta):
     Inherit from this class to pre-define mandatory components in the model.
     """
 
-    _DATADIR = ""  # path to the model data folder
     _NAME: str = "modelname"
-    _MAPS = {"<general_hydromt_name>": "<model_name>"}
-    _FOLDERS = [""]
```

Comment:
This is behavior that we are deleting so we should updtae the changelog / migration guide.
Any subfolders listed was generated automatically during model __init__ so for plugins, this may break some of their writing methods in case they do not create these subfolders.

Reply:
I've added it to the migration guide.